idunn/blocks/wikipedia.py

```diff
@@ -5,7 +5,63 @@
 from requests.exceptions import HTTPError, RequestException, Timeout
 import pybreaker
 from elasticsearch import Elasticsearch
+from redis import ConnectionPool, ConnectionError, TimeoutError
 
+from redis_rate_limit import RateLimiter, TooManyRequests
+
+class HTTPError40X(HTTPError):
+    pass
+
+class WikipediaLimiter:
+    _limiter = None
+    _redis_url = None
+    _redis_db = None
+
+    @classmethod
+    def init_limiter(cls):
```

Comment:
minor: it would seems more natural for this to be a lazy getter

Reply:
done